src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/MålgruppeValidering.kt

```diff
@@ -27,4 +34,96 @@ object MålgruppeValidering {
             "målgruppe=$målgruppeType er ikke gyldig for $stønadstype"
         }
     }
+
+    fun vurderAldersvilkår(
+        vilkårperiode: LagreVilkårperiode,
+        stønadstype: Stønadstype,
+        grunnlagsdata: Grunnlagsdata,
+    ): VurderingAldersVilkår {
+        val fødselsdato = grunnlagsdata.grunnlag.fødsel?.fødselsdato
+
+        feilHvis(fødselsdato == null) { "Kan ikke vurdere aldersvilkår uten å vite fødselsdato til bruker" }
+
+        val gyldig: SvarJaNei? =
+            when (stønadstype) {
+                Stønadstype.BARNETILSYN, Stønadstype.LÆREMIDLER ->
+                    when (vilkårperiode.type as MålgruppeType) {
+                        MålgruppeType.AAP, MålgruppeType.NEDSATT_ARBEIDSEVNE, MålgruppeType.UFØRETRYGD ->
+                            vurderAldersvilkårForNedsattArbeidsevne(fødselsdato, vilkårperiode)
+                        MålgruppeType.OMSTILLINGSSTØNAD -> vurderAldersvilkårForOmstillingsstønad(fødselsdato, vilkårperiode)
+                        MålgruppeType.OVERGANGSSTØNAD -> SvarJaNei.JA
+                        MålgruppeType.DAGPENGER -> null
+                        MålgruppeType.SYKEPENGER_100_PROSENT -> null
+                        MålgruppeType.INGEN_MÅLGRUPPE -> null
+                    }
+
+                Stønadstype.BOUTGIFTER -> TODO("Stønadstype BOUTGIFTER er ikke implementert")
+            }
+        feilHvis(gyldig == SvarJaNei.NEI) {
+            "Aldersvilkår er ikke oppfylt ved opprettelse av målgruppe=${vilkårperiode.type} for behandling=${grunnlagsdata.behandlingId}"
+        }
+
+        return VurderingAldersVilkår(
```

Comment:
Hvis man flytter valideringen til en egen fil, eller inn i `VurderingAldersVilkår` så kan det bli enklere å unitteste den. Tenker det hadde vært fint med noen boundry-tests for alderssjekkene

Reply:
Flyttet til egen fil: e90f682ab9d84b42a6f187220b5eccec55859ecb